react/components/Link.js

```diff
@@ -6,8 +6,11 @@ const isLeftClickEvent = event => event.button === 0
 const isModifiedEvent = event =>
   !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey)
 
-const createLocationDescriptor = (to, {query}) =>
-  query ? {pathname: to, search: query} : to
+const createLocationDescriptor = (to, {query}) => ({
+  pathname: to,
+  state: {renderRouting: true},
```

Comment:
Sera que esse `if` nao deveria ta do lado da @analuizamtg?

O default ser pelo Render e a Ana opt-out disso explicitamente.

Reply:
Ela não tem controle sobre o listener do render-runtime. O que coloquei pra controle foi uma variável `__RUNTIME__.customRouting`, que ela deve configurar na app dela. Se a navegação não foi feita usando o componente `<Link>` do runtime (isso é verificado neste `state`) e o `customRouting` for true, a navegação built-in é ignorada.